internal/cmd/kafka/command_cluster_update.go

```diff
@@ -155,3 +229,15 @@ func confirmShrink(prompt form.Prompt, promptMessage string) (bool, error) {
 	}
 	return true, nil
 }
+
+func getClusterType(config cmkv2.CmkV2ClusterSpecConfigOneOf) string {
+	if config.CmkV2Dedicated != nil {
+		return skuDedicated
+	}
+
+	if config.CmkV2Standard != nil {
+		return skuStandard
+	}
+
+	return skuBasic
+}
```

Comment:
It seems easier if we have all the switch statement functionality here and call this function anytime we need to get the cluster type. You would just need to add another case if the user has inputted an invalid type.